docs/website/mtouch-errors.md

```diff
@@ -1259,6 +1259,7 @@ An unexpected error occured when processing an assembly.
 
 The assembly causing the issue is named in the error message. In order to fix this issue the assembly will need to be provided in a [bug report](https://bugzilla.xamarin.com) along with a complete build log with verbosity enabled (i.e. `-v -v -v -v` in the **Additional mtouch arguments**).
 
+<!--- 2104 used by mmp -->
```

Comment:
That seems like a check that `mtouch` should also be doing

Reply:
I assumed no one would be crazy enough to recompile a mixed mode assembly for XI (since you can't just reuse a windows desktop lib like you can on macOS).

I can add the check if you'd like though.

Reply:
@rolfbjarne Thoughts on check? You gave 👍  before but I can add check and test if needed.

Reply:
@chamons netportable/pcl assemblies could be mixed-mode (without being re-compiled for XI)

Reply:
@chamons yeah, I thought about mtouch too, but reached the same conclusion, it's much more likely to run into this problem with Xamarin.Mac.

That said, @spouliot is right, in our new world of assembly sharing (netportable, pcl, ...) people might run into this on XI as well, and a good error message (which is easy to implement, as in this case), is IMHO worth it.